x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java

```diff
@@ -413,7 +414,10 @@ private static FunctionDefinition[][] functions() {
                 def(MvSum.class, MvSum::new, "mv_sum"),
                 def(Split.class, Split::new, "split") },
             // fulltext functions
-            new FunctionDefinition[] { def(Match.class, Match::new, "match"), def(QueryString.class, QueryString::new, "qstr") } };
+            new FunctionDefinition[] {
+                def(Match.class, Match::new, "match"),
+                def(QueryString.class, QueryString::new, "qstr"),
+                def(Term.class, Term::new, "term"), } };
```

Comment:
sorry I did not see this in the first review - this needs to be in `snapshotFunctions`

Reply:
after this is fixed it should be good to merge